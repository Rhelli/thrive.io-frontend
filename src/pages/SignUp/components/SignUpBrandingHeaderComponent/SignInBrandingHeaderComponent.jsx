import React from 'react';
import thriveLogo from '../../../../assets/img/thrive-full-transparent-alt.png';
import styles from './SignInBrandingHeaderComponent.module.scss';

const SignInBrandingHeaderComponent = () => (
  <header className={styles.brandingHeader}>
    <img src={thriveLogo} alt="Thrive" />
  </header>
);

export default SignInBrandingHeaderComponent;
