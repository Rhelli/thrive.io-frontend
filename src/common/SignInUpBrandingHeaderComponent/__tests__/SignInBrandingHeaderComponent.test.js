import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Enzyme, { mount } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import SignInBrandingHeaderComponent from '../SignInBrandingHeaderComponent';

Enzyme.configure({ adapter: new Adapter() });

const signInBrandingHeaderComponent = (
  <SignInBrandingHeaderComponent />
);

describe('The appearance and functionality of the component', () => {
  const container = mount(signInBrandingHeaderComponent);

  it('Matches the previous snapshot', () => {
    const tree = renderer.create(signInBrandingHeaderComponent).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Has the correct container name', () => {
    expect(container.childAt(0).hasClass('brandingHeader')).toBe(true);
  });

  it('Displays the thrive logo image', () => {
    expect(container.childAt(0).childAt(0).name()).toBe('img');
    expect(container.childAt(0).childAt(0).prop('src')).toBe('thrive-full-transparent-alt.png');
  });
});
