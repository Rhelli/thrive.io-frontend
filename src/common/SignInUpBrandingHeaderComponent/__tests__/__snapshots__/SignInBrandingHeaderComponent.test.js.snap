// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The appearance and functionality of the component Matches the previous snapshot 1`] = `
<header
  className="brandingHeader"
>
  <img
    alt="Thrive"
    src="thrive-full-transparent-alt.png"
  />
</header>
`;
